### ISSUE TITLE
Error: ENOENT: no such file or directory, open '/Users/noel/XXX/app/ios/XXX.xcodeproj/project.pbxproj'

### ISSUE THREAD

**Issue by BrodaNoel** [unimonkiez/react-native-asset#5]
Looks like `react-native-asset` is looking for `ios/XXX.xcodeproj/project.pbxproj'`.
I have not that file. My file is `ios/XXX.xcodeproj/project.xcworkspace`.

I'm using XCode 9

**Comment by unimonkiez** [comment]
You are right! currently only supports the simple projects which are ejected from react-native.  
Need a fix to support both
